executor/executor.go

```diff
@@ -2186,6 +2186,8 @@ func ResetContextOfStmt(ctx sessionctx.Context, s ast.StmtNode) (err error) {
 	vars.PrevFoundInBinding = vars.FoundInBinding
 	vars.FoundInBinding = false
 	vars.DurationWaitTS = 0
+	vars.CurrInsertBatchExtraCols = nil
```

Comment:
Why set to nil instead of resetting the old one?

Reply:
It cannot guarantee that the slice header here is not nil.